Unicode/is_Ligature.c

```diff
@@ -1,104 +1,21 @@
 /*
 Copyright: 2012 Barry Schwartz
+    Create is_Ligature.c (using a python script) to test for Vulgar fractions
 Copyright: 2016 Joe Da Silva
-License: BSD-3-clause
+    Re-write is_Ligature.c to test for ligatures, Vulgar and Other-fractions.
+Copyright: 2016 Gioele Barabucci
+    Simplify makeutype.c code. Split is_Ligature.c, create is_Ligature_data.h
+License: GPL-3+
 Contributions:
 */
 
-/* This file was generated using the program 'makeutype' for Unicode_version 9.0 */
+#ifndef FONTFORGE_UNICODE_IS_LIGATURE_C
```

Comment:
hi @frank-trampe, I do not know why that header guard is in that C file; it is not in any of my commits. I suppose it somehow got there while @JoesCat was mixing my patches with his work on Unicode 9. Weird things happen all the time with git. ;)